commands/eject.js

```diff
@@ -36,12 +36,12 @@ const internal = [
 ];
 const enhanced = ['chalk', 'cross-spawn', 'filesize', 'fs-extra', 'minimist', 'strip-ansi'];
 const content = ['@babel/polyfill', 'react', 'react-dom'];
-const bareDeps = {rimraf: '^2.6.2'};
+const bareDeps = {'cpy-cli': '^2.0.0', rimraf: '^2.6.2'};
 const bareTasks = {
 	serve: 'webpack-dev-server --hot --inline --env development --config config/webpack.config.js',
-	pack: 'webpack --env development --config config/webpack.config.js',
-	'pack-p': 'webpack --env production --config config/webpack.config.js',
-	watch: 'webpack --env development --config config/webpack.config.js --watch',
+	pack: 'webpack --env development --config config/webpack.config.js && cpy public dist',
```

Comment:
Is this supposed to reference PUBLIC_URL in some way?  Or, maybe I don't understand how it will be used below.

Reply:
It's just copying a `public` directory if it exists, into `dist`

Reply:
Maybe I should read what PUBLIC_URL is, first.   :disappear:

Reply:
PUBLIC_URL is used within apps via `process.env.PUBLIC_URL` for the url path to the output directory for the static assets.  For example, within jsx:
```
const myImg = () => (<img src={process.env.PUBLIC_URL + '/img.png'}/>);
```
The mainly is needed for scenarios where app  aren't hosted at root level, or assets go in different directories.  For usecases, Enact CLI sets `PUBLIC_URL` to `'.'` since the CLI isn't designed for other build-types. We mainly provide `process.env.PUBLIC_URL` to provide consistency for anyone coming from CRA :+1:

Reply:
👍 I read up on it a bit so I think I see now.